components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -27,7 +27,7 @@
     </chef-page-header>
     <section class="page-body" [hidden]="showTab('rules')">
       <form [formGroup]="projectForm">
-        <chef-form-field id="update-name">
+        <chef-form-field id="update-name" [watch]="projectForm.controls.name.value" [control]="projectForm.controls.name">
           <label>
             <span class="label">Name <span aria-hidden="true">*</span></span>
             <input chefInput formControlName="name" type="text"
```

Comment:
I don't know how `chefInput` works under the hood, but *only having to specify* `name` (not `projectForm.controls.name`) seems like a nice tweak. Can that be carried over somehow?

Reply:
Great question! `formControlName` is an Angular directive, not (directly) related to `input` or `chefInput`. From that Angular can easily find the parent form so it can divine `projectForm.control.name`... However, even if we could do the same machination, per my other comment `watch` needs the actual thing, not something that we can assemble from parts once we get to TypeScript. :shrug:

Reply:
Thanks! It's beyond me to assess, so I'll blindly trust you on this 😉